.gitignore

```diff
@@ -3,8 +3,10 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is typically used to store user-uploaded files in web applications. Ensure that this is intended to be ignored, as it might be necessary to keep track of these files in some cases, especially if they are not generated or can be restored easily.